packages/autorest.python/autorest/codegen/serializers/metadata_serializer.py

```diff
@@ -134,10 +152,14 @@ def _is_paging(operation):
                 self.code_model.client.imports_for_multiapi(async_mode=True).to_dict()
             ),
             sync_config_imports=_json_serialize_imports(
-                self.code_model.config.imports(async_mode=False).to_dict()
+                self.code_model.config.imports(
+                    async_mode=False, for_multiapi=True
```

Comment:
i think there has to be a better way than adding `for_multiapi`. Why are we adding this?

Reply:
The `type_annotation()` is never used in the multiapi files. So the imports for multiapi are different from the normal imports as we only import the "Literal" when it's not for multiapi.

Referring to existing pattern for `Client` at https://github.com/Azure/autorest.python/blob/20220922-constants/packages/autorest.python/autorest/codegen/models/client.py#L182, I updated to do this by a `Config.imports_for_multiapi()` function: https://github.com/Azure/autorest.python/pull/1464/commits/d4f15d82df6846840a15972bb4e1dba305986f6c.